src/content/docs/usage/prebuilds.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Prebuilds
+description: Learn how to manage and reuse Prebuilds across Workspaces.
+sidebar:
+  label: Prebuilds
+---
+
+Prebuilds are designed to speed up development by setting up a [project configuration](/usage/projects#project-configuration) that listens to changes in the connected Git Provider repository. This ensures a quicker project creation process when initiating a Workspace.
+
+Prebuilds work by registering a listener to webhook events from the Git Provider by sending a public API endpoint that the Git Provider will use to send requests.
```

Comment:
> Prebuilds work by registering a listener to webhook events from the Git Provider by sending a public API endpoint that the Git Provider will use to send requests.

Prebuilds work by registering a listener for webhook events from the Git provider. A public API endpoint is provided, which the Git provider uses to send these requests.